internal/api/v1/handlers/instance.go

```diff
@@ -101,9 +101,9 @@ func (h *InstanceHandler) DeleteInstance(c *fiber.Ctx) error {
 		})
 	}
 
-	if req.Name == "" {
+	if req.InstanceName == "" {
```

Comment:
add comment here that this is technically not required if each instance has a name, but for simplicity we require this as a safe guard and it prevents needing to iterate over all the instances.